app/controllers/api/base_controller.rb

```diff
@@ -1,12 +1,13 @@
 class Api::BaseController < ApplicationController
   skip_before_action :verify_authenticity_token
 
-  private
```

Comment:
Usually I see `private` used as separator for "action methods" (public - for example index, update, create) and "helper methods" (private, for example `find_rubygem_by_name`, `gem_name`). There's no need to remove it IMHO.

Reply:
I normally use `private` in the same way inside controllers.

Reply:
I don't agree but i will address this issue in a different pr